install.md

```diff
@@ -212,6 +212,65 @@ Building in a container is simpler, but more restrictive:
 $ make binary
 ```
 
+### Build and Install shell completion scripts
+
+Skopeo can create shell completion scripts for bash, zsh, fish and powershell.
+
+This places the scripts under the `completions` directory:
+```bash
+$ make completions
+```
+
+If you want to install the files to the system directories (on linux) use:
+```bash
+$ sudo make install-completions
+```
+
+By default the completion scripts will show extra descriptions as help:
+```
+$ skopeo l[TAB]
+list-tags  (List tags in the transport/repository specified by the SOURCE-IMAGE)  logout     (Logout of a container registry)
```

Comment:
(A new line before `logout`?)

Reply:
When you have enough width the shell will place them on the same line.

Reply:
Ah, my fault for not actually trying that.